app/models/hub_edos/student.rb

```diff
@@ -0,0 +1,32 @@
+module HubEdos
+  class Student
+    attr_accessor :terms_in_attendance, :academic_level
+
+    def initialize(uid)
+      @uid = uid
+      if uid.present?
+        @student_hash = HubEdos::V2::Student.new(user_id: uid).get
+      end
+    end
+
+    def get_max_terms_in_attendance
```

Comment:
Consider removing the `get_` from these method names

Reply:
done